src/common/reactNativeProjectHelper.ts

```diff
@@ -19,22 +18,44 @@ export class ReactNativeProjectHelper {
      * {operation} - a function that performs the expected operation
      */
     public isReactNativeProject(): Q.Promise<boolean> {
-        let currentPackage = new Package(this.workspaceRoot);
-        return currentPackage.dependencies().then(dependencies => {
-            return !!(dependencies && dependencies["react-native"]);
-        }).catch((err: Error) => {
-            return Q.resolve(false);
+        let deferred = Q.defer<any>();
+
+        exec("react-native -v", {
+            cwd: this.workspaceRoot,
+        }, (error, stdout, stderr) => {
+            if (error) {
+                deferred.reject(error);
+            }
+
+            let version = stdout.toString().match(/react-native: ([\d\.]+)/);
+            deferred.resolve(version && semver.gte(version[1], "0.19.0"));
```

Comment:
We shouldn't be validating a specific version in this method, this should only check if the project is valid or no (`react-native -v` returns `react-native: n/a - not inside a React Native project directory` if the project is not valid).